ethereumj-core/src/main/java/org/ethereum/core/BlockchainImpl.java

```diff
@@ -209,11 +223,6 @@ public BlockchainImpl withAdminInfo(AdminInfo adminInfo) {
         return this;
     }
 
-    public BlockchainImpl withEthereumListener(EthereumListener listener) {
```

Comment:
Listener doesn't look the thing which is required for running `BlockchainImpl`. Also old constructor could be used by somebody. I'd prefer to keep both old constructor (and init listener there with `EthereumListener.EMPTY`) and `withEthereumListener` method.